python/dgl/graphbolt/dataset.py

```diff
@@ -117,18 +122,21 @@ class OnDiskDataset(Dataset):
     def __init__(self, path: str) -> None:
         with open(path, "r") as f:
             self._meta = OnDiskMetaData.parse_raw(f.read(), proto="yaml")
+        self._train_set = self._init_tvt_sets(self._meta.train_set)
+        self._validation_set = self._init_tvt_sets(self._meta.validation_set)
+        self._test_set = self._init_tvt_sets(self._meta.test_set)
 
     def train_set(self) -> ItemSet or ItemSetDict:
```

Comment:
This says train_set is ItemSet or ItemSetDict, but _init_tvt_sets returns List[ItemSet] or List[ItemSetDict].